src/components/MovieDetails/index.tsx

```diff
@@ -112,6 +112,31 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
   useEffect(() => {
     setShowManager(router.query.manage == '1' ? true : false);
   }, [router.query.manage]);
+  const [deviceUrl, setDeviceUrl] = useState(data?.mediaInfo?.plexUrl);
+  const [deviceUrl4k, setDeviceUrl4k] = useState(data?.mediaInfo?.plexUrl4k);
```

Comment:
```suggestion
  const [plexUrl, setPlexUrl] = useState(data?.mediaInfo?.plexUrl);
  const [plexUrl4k, setPlexUrl4k] = useState(data?.mediaInfo?.plexUrl4k);
```
I think it's best to call this `plexUrl` since we may support additional media servers in the future, in which case a media item could have a Plex URL in addition to a Jellyfin URL, etc.

Reply:
Updated.